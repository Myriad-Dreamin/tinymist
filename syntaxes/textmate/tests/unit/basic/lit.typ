#1.1em
#1.em
#.1em
#1.1%
#1.%
#.1%
#1.1
#1.
#.1
#1.1e1
#1
#11
#11.1
#11.11
#(1.11)
#(11.1)