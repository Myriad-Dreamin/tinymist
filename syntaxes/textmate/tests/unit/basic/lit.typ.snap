>#1.1em
#^ source.typst constant.numeric.hash.typst
# ^^^^^ source.typst constant.numeric.length.typst
>#1.em
#^ source.typst constant.numeric.hash.typst
# ^ source.typst constant.numeric.float.typst
#  ^ source.typst keyword.operator.accessor.typst
#   ^^ source.typst variable.other.readwrite.typst
>#.1em
#^ source.typst keyword.control.hash.typst
# ^ source.typst keyword.operator.accessor.typst
#  ^^^ source.typst constant.numeric.length.typst
>#1.1%
#^ source.typst constant.numeric.hash.typst
# ^^^^ source.typst constant.numeric.percentage.typst
>#1.%
#^ source.typst constant.numeric.hash.typst
# ^^^ source.typst constant.numeric.percentage.typst
>#.1%
#^ source.typst keyword.control.hash.typst
# ^ source.typst keyword.operator.accessor.typst
#  ^^ source.typst constant.numeric.percentage.typst
>#1.1
#^ source.typst constant.numeric.hash.typst
# ^^^ source.typst constant.numeric.float.typst
>#1.
#^ source.typst constant.numeric.hash.typst
# ^ source.typst constant.numeric.float.typst
#  ^ source.typst keyword.operator.accessor.typst
>#.1
#^ source.typst keyword.control.hash.typst
# ^ source.typst keyword.operator.accessor.typst
#  ^ source.typst constant.numeric.integer.typst
>#1.1e1
#^ source.typst constant.numeric.hash.typst
# ^^^^^ source.typst constant.numeric.float.typst
>#1
#^ source.typst constant.numeric.hash.typst
# ^ source.typst constant.numeric.integer.typst
>#11
#^ source.typst constant.numeric.hash.typst
# ^^ source.typst constant.numeric.integer.typst
>#11.1
#^ source.typst constant.numeric.hash.typst
# ^^^^ source.typst constant.numeric.float.typst
>#11.11
#^ source.typst constant.numeric.hash.typst
# ^^^^^ source.typst constant.numeric.float.typst
>#(1.11)
#^ source.typst keyword.control.hash.typst
# ^ source.typst meta.brace.round.typst
#  ^^^^ source.typst constant.numeric.float.typst
#      ^ source.typst meta.brace.round.typst
>#(11.1)
#^ source.typst keyword.control.hash.typst
# ^ source.typst meta.brace.round.typst
#  ^^^^ source.typst constant.numeric.float.typst
#      ^ source.typst meta.brace.round.typst